# HG changeset patch
# User Rik <rik@octave.org>
# Date 1401831720 25200
#      Tue Jun 03 14:42:00 2014 -0700
# Branch stable
# Node ID 96f22d6674c4c88f95e5933370ee538cfed025cd
# Parent  322eb69e30ad76e913d5d3b8bac2cb4312456757
Fix incorrect sparse matrix example code in oct-file chapter (bug #41799).

* external.txi: Use 1-based indexing in sparse matrix creation example to match
SparseMatrix constructor.

diff --git a/doc/interpreter/external.txi b/doc/interpreter/external.txi
--- a/doc/interpreter/external.txi
+++ b/doc/interpreter/external.txi
@@ -677,20 +677,20 @@ technique
 @group
 int nz, nr, nc;
 nz = 4, nr = 3, nc = 4;
 
 ColumnVector ridx (nz);
 ColumnVector cidx (nz);
 ColumnVector data (nz);
 
-ridx(0) = 0; cidx(0) = 0; data(0) = 1; 
-ridx(1) = 0; cidx(1) = 1; data(1) = 2; 
-ridx(2) = 1; cidx(2) = 3; data(2) = 3; 
-ridx(3) = 2; cidx(3) = 3; data(3) = 4;
+ridx(0) = 1; cidx(0) = 1; data(0) = 1;
+ridx(1) = 2; cidx(1) = 2; data(1) = 2;
+ridx(2) = 2; cidx(2) = 4; data(2) = 3;
+ridx(3) = 3; cidx(3) = 4; data(3) = 4;
 SparseMatrix sm (data, ridx, cidx, nr, nc);
 @end group
 @end example
 
 @noindent
 which creates the matrix given in section
 @ref{Storage of Sparse Matrices}.  Note that the compressed matrix
 format is not used at the time of the creation of the matrix itself,
